### ISSUE TITLE
resource-deleter cannot be run due the 'ReferenceError: regeneratorRuntime is not defined'

### ISSUE THREAD

**Issue by peniakoff** [commercetools/nodejs#1768]
### Description

After last update cannot run `resource-deleter` application due the `ReferenceError: regeneratorRuntime is not defined`.

### Context

NodeJS: v14.19.0
npm: 8.5.1

**Comment by d-kretschmann-dotsource** [comment]
We ran into the same issue on the resource-deleter version 2.0.26. There seems to be no issue when you revert to the previous version:
```
npm install -g @commercetools/resource-deleter@2.0.25
```

**Comment by peniakoff** [comment]
True, @d-kretschmann-dotsource, but it will be great to have opportunity to run the newest version ofc.
